victor_description/urdf/victor.urdf.xacro

```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0"?>
 <robot name="victor" xmlns:xacro="http://www.ros.org/wiki/xacro">
+    <xacro:arg name="use_tools" default="true"/>
```

Comment:
I would change this arg to `tool_name`.
If possible use logic like:
`<xacro:if value=($tool_name) != "":...`

If xacro does not support this, then keep the `use_tools` and add a `tool_name` arg

Reply:
I think this depends on usage. Do we want the user to be able to choose between a number of different transforms, any of which will get the generic name "left_tool", or do we want any number of different tools with different frames? In the latter case, I see no reason to have any xacro conditionals, just always include all the tools. I think the all-tools-all-the-time approach makes the most sense. Especially since you might have multiple frames you care about within the same task.

Reply:
plus, I do not foresee us defining very many tools